pkg/config/types.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+const (
+	// counter - attacher package
+	CPUCycle       = "cpu_cycles"
+	CPUInstruction = "cpu_instr"
+	CacheMiss      = "cache_miss"
+	// bpf - attacher package
+	CPUTime = "cpu_time"
+	// cgroup - cgroup package
+	CgroupfsMemory       = "cgroupfs_memory_usage_bytes"
```

Comment:
Can we have the metric source prefix only `cgroup` instead of `cgrpupfs`?
wdyt?

Reply:
save as above. Let's rename all of these in other PR.